docs/dev/release.rst

```diff
@@ -214,34 +267,99 @@ You can create a clean checkout as follows (requires
 #. Check `Zenodo page for Jdaviz <https://doi.org/10.5281/zenodo.5513927>`_.
    It should have picked up the GitHub Release automatically.
 
+#. The release is basically done, but now you have to set it up for the
+   *next* release cycle. Checkout the ``main`` branch and update ``CHANGES.rst``
+   using your admin power. If you do not have sufficient access to do that,
+   you will have to update it via a pull request from your fork. Make sure the
+   section for the version just released matches the finalized change log from
+   the release branch (be sure to change ``unreleased`` to the appropriate date),
+   and add a new bugfix release section below the next feature
+   release section as follows, replacing ``X.Y.Z`` with the next minor release
+   number. For example, if you just released ``3.0.2``, a section for ``3.0.3``
+   would go below the section for ``3.1``::
+
+     X.Y.Z (unreleased)
+     ==================
+
+     Bug Fixes
+     ---------
+
+     Cubeviz
+     ^^^^^^^
+
+     Imviz
+     ^^^^^
+
+     Mosviz
+     ^^^^^^
+
+     Specviz
+     ^^^^^^^
+
+     Specviz2d
+     ^^^^^^^^^
+
+#. Commit your changes of the, uh, change log with a message, "Back to development: A.B.dev"
+
 #. Follow procedures for :ref:`release-milestones`.
 
+#. For your own sanity unrelated to the release, grab the new tag for your fork::
+
+     git fetch upstream --tags
+
 Congratulations, you have just released a new version of Jdaviz!
 
+.. _release-milestones:
 
-.. _release-short:
+Milestones bookkeeping
+======================
 
-The short way
-=============
+#. Go to `Milestones <https://github.com/spacetelescope/jdaviz/milestones>`_.
 
-This way is for when you are in a rush, are very familiar with the process already,
-and are deploying on a proven automated process. It is faster but also has a higher
-risk factor. If you choose this way wrongly, you will end up doing hotfix releases
-anyway, which will not save you any time in the end. Only go this way if you know
-what you are doing.
+#. Create a new milestone for the next release and the next bugfix release, if
+   doing a feature release, or for just the next bugfix release if you just did
+   one.
 
-You can do a release from your fork directly without a clean code check-out.
+#. For the milestone of this release, if there are any open issues or pull requests
+   still milestoned to it, move their milestones to the next feature or bugfix
+   milestone as appropriate.
+
+#. Make sure the milestone of this release ends up with "0 open" and then close it.
+
+#. Remind the other devs of the open pull requests with milestone moved that they
+   will need to move their change log entries to the new release section that you
+   have created in ``CHANGES.rst`` during the release process.
+
+
+.. _release-old:
+
+The old, long way
```

Comment:
Do we really need this section? If we have to, we can always checkout the old copy under version control.

Reply:
Ah, I see you refer to this section above. Alternately, you can also generate a permalink to the old ways on GitHub and just link to that and remove this section.

Example permalink: https://github.com/spacetelescope/jdaviz/blob/a64f1a50a4ef26d92e904bfdf6e9cafcef84dbbc/docs/dev/release.rst